AEMG/training.py

```diff
@@ -143,11 +146,10 @@ def train(self, epochs=1000, patience=50, weight=[1,1,1,0]):
                 epoch_train_loss += loss_total.item()
 
             if weight[3] != 0:
-                for i, (x_final, label) in enumerate(self.labels_loader):
+                for i, (triplets, x_final) in enumerate(self.labels_loader):
```

Comment:
Would `x_final` be returned given the above changes?

Reply:
Yes, I have a collate_fn that will return x_final